core/services/pipeline/common.go

```diff
@@ -74,6 +74,34 @@ const (
 	InputTaskKey = "input"
 )
 
+// RunInfo contains additional information about the finished TaskRun
+type RunInfo struct {
```

Comment:
Let's just call this pipeline.Meta?

Reply:
It got confusing because we have other things called `Meta`, that's why I used the name `RunInfo` instead so it is unambiguous and clear.